Prose/Prose.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -63,6 +63,15 @@
         "version" : "0.4.0"
       }
     },
+    {
+      "identity" : "swift-tagged",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/pointfreeco/swift-tagged",
+      "state" : {
+        "revision" : "f3f773a5e13f3c8f0ab1ce2ae6378058acefa87d",
+        "version" : "0.7.0"
+      }
+    },
```

Comment:
I didn't know this library!
(It's only for prefixed types but I created https://github.com/RemiBardon/swift-prefixed-type, if one day you need this kind of stuff)